src/govuk/all.mjs

```diff
@@ -101,5 +102,6 @@ export {
   NotificationBanner,
   Radios,
   SkipLink,
-  Tabs
+  Tabs,
+  I18n
```

Comment:
If I'm reading this right, I think this is a related but separate change which makes i18n available to folks importing GOV.UK Frontend, and also makes it available at `window.GOVUKFrontend.I18n`?

I think this makes sense but it feels significant enough that it's worth splitting this and the corresponding test changes out into a separate commit and mentioning in the PR description.

Reply:
Sounds good, mind taking a look now?